src/index.js

```diff
@@ -0,0 +1,42 @@
+import _ from "lodash";
+import "./style.css";
+
+const taskArray = [
+  {
+    index: 0,
+    descriptions: "I  will code",
+    completed: true,
+  },
+  {
+    index: 1,
+    descriptions: "i love code",
+    completed: false,
+  },
+  {
+    index: 2,
+    descriptions: "i like webpack",
+    completed: true,
+  },
+  {
+    index: 4,
+    description: "I love microverse",
+    completed: false,
+  },
+];
+
+const list = document.getElementById("list");
+
+const createList = () => {
+  const sortedArray = taskArray.sort((a, b) => a.index - b.index);
+  sortedArray.forEach((task) => {
+    const eachTask = document.createElement("li");
+    eachTask.innerHTML = `<ul id="eachTask">
+        <ul><input type="checkbox" name=" " class="task">
+        <label for="">${task.descriptions}</label></div>
+        <i class="fa fa-ellipsis-v"></i>
+        </ul><hr>`;
+    list.appendChild(eachTask);
+  });
```

Comment:
- [ ] You are required to match the wireframe given. Kindly improve your styling to do that.

- [ ] You also have a task that displays `undefined`. Kindly fix that

  Your design | Required layout
  :---:|:---:
  ![fnb](https://user-images.githubusercontent.com/54848363/195308212-cd1373c3-db68-4bca-8b18-653a69baab19.jpg) | ![fmfm](https://user-images.githubusercontent.com/54848363/195308204-e451cd42-f219-42ac-ae08-eb77b2318b78.jpg)